package.json

```diff
@@ -42,22 +42,22 @@
     "eslint-plugin-react": "=5.1.1",
     "eslint-plugin-react-native": "=1.0.2",
     "mocha": "=2.5.3",
-    "react": "=15.1.0",
-    "react-native": "^0.27.0"
+    "react": "=15.4.0",
+    "react-native": ">=0.27.0"
   },
   "dependencies": {
-    "cubic-bezier": "=0.1.2",
-    "invariant": "=2.2.1",
-    "keymirror": "=0.1.1",
-    "raf": "=3.2.0",
-    "react-addons-create-fragment": "=15.1.0",
-    "react-addons-linked-state-mixin": "=15.1.0",
-    "react-addons-perf": "=15.1.0",
-    "react-addons-pure-render-mixin": "=15.1.0",
-    "react-addons-test-utils": "=15.1.0",
-    "react-addons-update": "=15.1.0",
-    "react-timer-mixin": "=0.13.3",
-    "warning": "=2.1.0"
+    "cubic-bezier": "^0.1.2",
+    "invariant": "^2.2.1",
+    "keymirror": "^0.1.1",
+    "raf": "^3.2.0",
+    "react-addons-create-fragment": "^15.4.0",
+    "react-addons-perf": "^15.4.0",
+    "react-addons-pure-render-mixin": "^15.4.0",
+    "react-addons-test-utils": "^15.4.0",
+    "react-addons-update": "^15.4.0",
+    "react-dom": "^15.4.0",
```

Comment:
do we need to explicitly state a dependancy on react-dom? Surely the addons libraries will handle installing this for us?